templates/genai-perf-templates/tutorial_template

```diff
@@ -28,192 +28,121 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Profile Large Language Models with GenAI-Perf
 
-- [Profile GPT2 running on Triton + TensorRT-LLM](#tensorrt-llm)
-- [Profile GPT2 running on Triton + vLLM](#triton-vllm)
-- [Profile GPT2 running on OpenAI Chat Completions API-Compatible Server](#openai-chat)
-- [Profile GPT2 running on OpenAI Completions API-Compatible Server](#openai-completions)
-
----
-
-## Profile GPT2 running on Triton + TensorRT-LLM <a id="tensorrt-llm"></a>
-
-### Run GPT2 on Triton Inference Server using TensorRT-LLM
-
-<details>
-<summary>See instructions</summary>
-
-Run Triton Inference Server with TensorRT-LLM backend container:
+This tutorial will demonstrate how you can use GenAI-Perf to measure the performance of
+various inference endpoints such as
+[KServe inference protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2)
+and [OpenAI API](https://platform.openai.com/docs/api-reference/introduction)
+that are widely used across the industry.
 
-```bash
-export RELEASE="{{ release }}"
+### Table of Contents
 
-docker run -it --net=host --gpus=all --shm-size=2g --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/tritonserver:${RELEASE}-trtllm-python-py3
+- [Profile GPT2 running on Triton + TensorRT-LLM Backend](#tensorrt-llm)
+- [Profile GPT2 running on Triton + vLLM Backend](#triton-vllm)
+- [Profile GPT2 running on OpenAI Chat Completions API-Compatible Server](#openai-chat)
+- [Profile GPT2 running on OpenAI Completions API-Compatible Server](#openai-completions)
 
-# Install Triton CLI (~5 min):
-pip install "git+https://github.com/triton-inference-server/triton_cli@{{ triton_cli_version }}"
+</br>
 
-# Download model:
-triton import -m gpt2 --backend tensorrtllm
+## Profile GPT-2 running on Triton + TensorRT-LLM <a id="tensorrt-llm"></a>
 
-# Run server:
-triton start
-```
-
-</details>
+You can follow the [quickstart guide](https://github.com/triton-inference-server/triton_cli?tab=readme-ov-file#serving-a-trt-llm-model)
+in the Triton CLI Github repository to serve GPT-2 on the Triton server with the TensorRT-LLM backend.
 
 ### Run GenAI-Perf
 
-Run GenAI-Perf from Triton Inference Server SDK container:
+Run GenAI-Perf inside the Triton Inference Server SDK container:
 
 ```bash
-export RELEASE="{{ release }}"
-
-docker run -it --net=host --gpus=all nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
```

Comment:
@nv-hwoo @debermudez @dyastremsky why was the SDK container run command removed from the tutorial?

I think generally a tutorial user is going to be unfamiliar, so we should give them as much copy-pasteable commands as possible so they don't get stuck.

Reply:
I don't know. The [GenAi-Perf README](https://github.com/triton-inference-server/perf_analyzer/tree/main/genai-perf) covers [installation](https://github.com/triton-inference-server/perf_analyzer/tree/main/genai-perf#installation) and these are supposed to be additional examples, so it's possible Hyunjae wanted to remove redundancy.

 Hyunjae will be back in a couple of weeks. We could look at whether these need to be re-added then.